GUI/MainInstall.cs

```diff
@@ -434,6 +434,7 @@ private void UpdateRecommendedDialog(Dictionary<string, List<string>> mods, bool
                     "The following modules have been recommended by one or more of the chosen modules:";
                 RecommendedModsListView.Columns[1].Text = "Recommended by:";
                 RecommendedModsToggleCheckbox.Text = "(De-)select all recommended mods.";
+                RecommendedModsToggleCheckbox.Checked=true;
```

Comment:
Shot in the dark here, can I fix that by setting a similar but false line after the else switch below?